spec/liquid_tags/user_tag_spec.rb

```diff
@@ -21,9 +21,11 @@ def generate_user_tag(id_code)
     end
   end
 
-  it "rejects invalid id_code" do
-    expect do
-      generate_user_tag("this should fail")
-    end.to raise_error(StandardError)
+  context "when given an invalid id_code" do
```

Comment:
I think this is leftover copy/paste from the comment specs. Can you change this to `username` rather than `id_code`?